# Description (restore-spies)

The `restore-spies` rule enforces the developer to use `restoreMock()` function that spied objects in unit testing.


## Rule Details


Examples of **incorrect** code for this rule:

```js

// fill me in

```

Examples of **correct** code for this rule:

```js

// fill me in

```


## When Not To Use It

You don't need to use that rule if you are the good one to restore your mocks or spies in your daily base or if you have been configured jest to [restoreMocks](https://jestjs.io/docs/en/configuration.html#restoremocks-boolean) to true 
